tests/_support/ea-server-mocker/js/ea-mocker.js

```diff
@@ -0,0 +1,71 @@
+(function ( $, undefined ) {
+	var formatJSONString = function ( text ) {
```

Comment:
My only nitpick in this file is that we use snake_case for our JS vars and function names.